src/components/Book/Book.js

```diff
@@ -1,33 +1,89 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux/es/exports';
-import { removeBook, remove } from '../../redux/books/books';
+import { removeBookApi } from '../../redux/books/apiActions';
 
-const Book = ({ title, author, id }) => {
+const Book = ({ ID, book }) => {
   const dispatch = useDispatch();
+  const {
+    title, author, category,
+  } = book;
   const removeHandler = () => {
-    dispatch(removeBook(remove(id)));
+    dispatch(removeBookApi(ID));
   };
+  const percent = Math.floor(Math.random() * (67 - 58 + 1)) + 56;
   return (
     <>
-      <div className="book">
-        <h2 className="title">{title}</h2>
-        <p className="author">{author}</p>
-        <button
-          type="button"
-          onClick={removeHandler}
-          className="remove"
-        >
-          Remove
-        </button>
+      <div className="book-panel">
+        <div>
+          <div className="book-info">
+            <p className="category">{category}</p>
+            <h2 className="title-1">{title}</h2>
+            <p className="author-1">{author}</p>
+          </div>
+          <div className="book-action-btns">
+            <button
+              type="button"
+              className="action-btn"
+            >
+              Comments
+            </button>
+            |
+            <button
+              type="button"
+              onClick={removeHandler}
+              className="action-btn"
+            >
+              Remove
+            </button>
+            |
+            <button
+              type="button"
+              className="action-btn"
+            >
+              Edit
+            </button>
+          </div>
+        </div>
+        <div className="progress">
+          <div className="oval-comp-wrap">
+            <div className="Rectangle-3">
+              <div className="Oval-2" />
+            </div>
+            <div className="percent-complete-wrap">
+              <div className="Lesson-Panel" />
+              <span className="-Percent-Complete">
+                {percent}
+                %
+              </span>
+              <span className="Completed">
+                Completed
+              </span>
+            </div>
+          </div>
+        </div>
+        <div className="chapter-inf-wrap">
+          <span className="Current-Chapter">
+            CURRENT CHAPTER
+          </span>
+          <span className="Current-Lesson">
+            Chapter3: A Lesson Learned
+          </span>
+          <div className="Rectangle-2">
+            <span className="Update-progress">
+              UPDATE PROGRESS
+            </span>
+          </div>
+        </div>
       </div>
     </>
   );
 };
```

Comment:
- [ ] **Required:** Kindly consider the following:

Your Design | Template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/167185680-e616b1a7-928b-4f64-aaf9-8e24c54d5526.png) | ![image](https://user-images.githubusercontent.com/27709832/167185449-5263dc49-0403-430f-8fa0-3771a669ccac.png)

Work on the following to have a similar design to the template.
- Spacing and padding.
- Colors.
- Font and font sizes.